booktrader/src/Component/Slideshow/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import {
+  Carousel,
+  CarouselItem,
+  CarouselControl,
+  CarouselIndicators
+} from "reactstrap";
+import book from "../../assets/book1.png";
+import styled from "styled-components";
+
+const Styles = styled.div`
+  .carousel {
+    margin-top: 50px;
+    margin-bottom: 50px;
+    border: none;
+  }
+  .carousel-control-prev-icon,
+  .carousel-control-next-icon {
+    height: 50px;
+    width: 50px;
+    filter: invert(100%);
+  }
+`;
+const items = [
+  {
+    src: `${book}`
+  },
+  {
+    src: `${book}`
+  },
+  {
+    src: `${book}`
+  }
+];
+
+const Slideshow = props => {
+  const [activeIndex, setActiveIndex] = useState(0);
+  const [animating, setAnimating] = useState(false);
+
+  const next = () => {
+    if (animating) return;
+    const nextIndex = activeIndex === items.length - 1 ? 0 : activeIndex + 1;
+    setActiveIndex(nextIndex);
+  };
+
+  const previous = () => {
+    if (animating) return;
+    const nextIndex = activeIndex === 0 ? items.length - 1 : activeIndex - 1;
+    setActiveIndex(nextIndex);
+  };
+  const goToIndex = newIndex => {
+    if (animating) return;
+    setActiveIndex(newIndex);
+  };
+
+  const slides = items.map(item => {
+    return (
+      <CarouselItem

```

Comment:
You must have index key for this because if you do not have this, console will have warning and make web to slow